src/Core/src/Core.csproj

```diff
@@ -74,7 +74,7 @@
 
   <!-- TypeScript compilation -->
   <PropertyGroup>
-    <TypeScriptTarget>ES2020</TypeScriptTarget>
+    <TypeScriptTarget>ES2015</TypeScriptTarget>
```

Comment:
Old ES for old devices.

Based on the failures of CI, only Android 30+ comes with a recent enough JS engine in Chrome. So until we increase the mi supported version on Android, we need this older code.

Reply:
Yeah, I was wrong but hoot I have no idea.